tests/test0183.d

```diff
@@ -0,0 +1,10 @@
+//T compiles:yes
+//T has-passed:yes
+//T retval:42
+
+int main() {
+	int a = mixin("-12");
```

Comment:
Please use more complex expressions. Have it appear in trickier places. Make sure it tests operator priority, rvalueness and all.